### ISSUE TITLE
Syncing so frequently

### ISSUE THREAD

**Issue by hkaancaliskan** [hwittenborn/celeste#197]
I have 2 file in the folder and nothing else. Running on Fedora 39 via flatpak.

[Screencast from 2024-02-12 00-31-10.webm](https://github.com/hwittenborn/celeste/assets/17825767/de577b08-eef5-49bf-923c-ba13d077cb43)

**Comment by primeroz** [comment]
Was wondering the same. 

I have installed Celeste on my `in-laws` laptop to sync their documents to google drive and made it auto-startup so that it could keep always in sync ... but it is constantly syncing which is not ideal .

An option to define how often to sync woult be great :pray:

**Comment by eriolloan** [comment]
> it is constantly syncing which is not ideal.

...especially on a laptop.

![image](https://github.com/hwittenborn/celeste/assets/15250783/ca5b4757-be9e-4c36-ae3d-553cfbe9ee2a)

**Comment by mateimm** [comment]
I also feel the need to either schedule the sync or option to pause it. Would be a nice-to-have thing. Thank you.

**Comment by jobdiogenes** [comment]
Ideally syncing will occurs  when file changes

**Comment by kenhaines2** [comment]
I also agree it syncs non-stop. I looked everywhere to try and find a config file but I guess it's hard coded??? It would be really nice if we could specify when it syncs for each entry. Also if you could have a setting where we could turn syncs off all together and have a "Manual Sync" button in the system tray drop-down. That would be perfect, id rather manual sync it then it run on a loop anyway(this would be better for some of us on laptops). Also the comment above mine for just file changes would be ok as well.

I found this not to long ago and I really like your work, thank you so much for making something like this.
